modules/virt-edit-cdrom-vm.adoc

```diff
@@ -9,37 +9,33 @@
 ifeval::["{context}" == "virt-edit-vms"]
 :object: virtual machine
 :object-gui: Virtual Machines
+:object-overview: Virtual Machine Overview
 endif::[]
 
 ifeval::["{context}" == "virt-editing-vm-template"]
 :object: virtual machine template
 :object-gui: Templates
+:object-overview: VM Template Details
 endif::[]
 
 [id="virt-vm-edit-cdrom_{context}"]
 
 = Editing CD-ROMs for {object-gui}
-Use the following procedure to configure CD-ROMs for virtual machines.
+Use the following procedure to edit CD-ROMs for {object}s.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-. Select the *Overview* tab.
+. Select a *{object-gui}* tab to open the *{object-overview}* screen.
```

Comment:
Re-posting my earlier comment about step 2 

> The steps should be:
> `. Click the *{object-gui}* tab`.
> `. Select a {object} to open the *{object-overview}* screen.`

Reply:
Content updated.

Reply:
From the .adoc file:

[id="virt-vm-edit-cdrom_{context}"]

= Editing CD-ROMs for {object-gui}
Use the following procedure to edit CD-ROMs for {object}s.

.Procedure

. Click *Workloads* -> *Virtualization* from the side menu.

. Click the {object-gui} tab.

. Select a *{object}* to open the *{object-overview}* screen.